### PR TITLE
LOC CHECKIN | dotnet/project-system master | 20190118

### PR DESCRIPTION
**Pull request by v-zbsail**

### CODE REVIEW

**Comment by janbaltus** [comment]
@davkean
regarding untranslated strings this will be covered during next cycle.
Please point me to xlm not valid file.

**Comment by jmarolf** [comment]
`src/Microsoft.VisualStudio.Editors/PropPages/xlf/ReferencePropPage.fr.xlf` appears to be completely invalid

`System.Xml.XmlException: Unexpected end of file has occurred. The following elements are not closed: trans-unit, body, file, xliff. Line 98, position 6.`

```ini
2019-01-18T12:58:14.2354938Z C:\Users\VssAdministrator\.nuget\packages\xlifftasks\0.2.0-beta-000081\build\XliffTasks.targets(90,5): error : 'ProjectSystem\Rules\xlf\PackageReference.xaml.cs.xlf' is out-of-date with 'ProjectSystem\Rules\PackageReference.xaml'. Run `msbuild /t:UpdateXlf` to update .xlf files or set UpdateXlfOnBuild=true to update them on every build, but note that it is strongly discouraged to set UpdateXlfOnBuild=true in official/CI build environments as they should not modify source code during the build. [D:\a\1\s\src\Microsoft.VisualStudio.ProjectSystem.Managed\Microsoft.VisualStudio.ProjectSystem.Managed.csproj]
2019-01-18T12:58:14.2355087Z C:\Users\VssAdministrator\.nuget\packages\xlifftasks\0.2.0-beta-000081\build\XliffTasks.targets(90,5): error MSB4018: The "UpdateXlf" task failed unexpectedly. [D:\a\1\s\src\Microsoft.VisualStudio.Editors\Microsoft.VisualStudio.Editors.vbproj]
2019-01-18T12:58:14.2355168Z C:\Users\VssAdministrator\.nuget\packages\xlifftasks\0.2.0-beta-000081\build\XliffTasks.targets(90,5): error MSB4018: System.Xml.XmlException: Unexpected end of file has occurred. The following elements are not closed: trans-unit, body, file, xliff. Line 98, position 6. [D:\a\1\s\src\Microsoft.VisualStudio.Editors\Microsoft.VisualStudio.Editors.vbproj]
2019-01-18T12:58:14.2355273Z C:\Users\VssAdministrator\.nuget\packages\xlifftasks\0.2.0-beta-000081\build\XliffTasks.targets(90,5): error MSB4018:    at System.Xml.XmlTextReaderImpl.Throw(Exception e) [D:\a\1\s\src\Microsoft.VisualStudio.Editors\Microsoft.VisualStudio.Editors.vbproj]
2019-01-18T12:58:14.2355351Z C:\Users\VssAdministrator\.nuget\packages\xlifftasks\0.2.0-beta-000081\build\XliffTasks.targets(90,5): error MSB4018:    at System.Xml.XmlTextReaderImpl.Throw(String res, String arg) [D:\a\1\s\src\Microsoft.VisualStudio.Editors\Microsoft.VisualStudio.Editors.vbproj]
2019-01-18T12:58:14.2355444Z C:\Users\VssAdministrator\.nuget\packages\xlifftasks\0.2.0-beta-000081\build\XliffTasks.targets(90,5): error MSB4018:    at System.Xml.XmlTextReaderImpl.ThrowUnclosedElements() [D:\a\1\s\src\Microsoft.VisualStudio.Editors\Microsoft.VisualStudio.Editors.vbproj]
2019-01-18T12:58:14.2355537Z C:\Users\VssAdministrator\.nuget\packages\xlifftasks\0.2.0-beta-000081\build\XliffTasks.targets(90,5): error MSB4018:    at System.Xml.XmlTextReaderImpl.ParseElementContent() [D:\a\1\s\src\Microsoft.VisualStudio.Editors\Microsoft.VisualStudio.Editors.vbproj]
2019-01-18T12:58:14.2355771Z C:\Users\VssAdministrator\.nuget\packages\xlifftasks\0.2.0-beta-000081\build\XliffTasks.targets(90,5): error MSB4018:    at System.Xml.XmlTextReaderImpl.Read() [D:\a\1\s\src\Microsoft.VisualStudio.Editors\Microsoft.VisualStudio.Editors.vbproj]
2019-01-18T12:58:14.2355941Z C:\Users\VssAdministrator\.nuget\packages\xlifftasks\0.2.0-beta-000081\build\XliffTasks.targets(90,5): error MSB4018:    at System.Xml.Linq.XContainer.ReadContentFrom(XmlReader r) [D:\a\1\s\src\Microsoft.VisualStudio.Editors\Microsoft.VisualStudio.Editors.vbproj]
2019-01-18T12:58:14.2356017Z C:\Users\VssAdministrator\.nuget\packages\xlifftasks\0.2.0-beta-000081\build\XliffTasks.targets(90,5): error MSB4018:    at System.Xml.Linq.XContainer.ReadContentFrom(XmlReader r, LoadOptions o) [D:\a\1\s\src\Microsoft.VisualStudio.Editors\Microsoft.VisualStudio.Editors.vbproj]
2019-01-18T12:58:14.2356119Z C:\Users\VssAdministrator\.nuget\packages\xlifftasks\0.2.0-beta-000081\build\XliffTasks.targets(90,5): error MSB4018:    at System.Xml.Linq.XDocument.Load(XmlReader reader, LoadOptions options) [D:\a\1\s\src\Microsoft.VisualStudio.Editors\Microsoft.VisualStudio.Editors.vbproj]
2019-01-18T12:58:14.2356199Z C:\Users\VssAdministrator\.nuget\packages\xlifftasks\0.2.0-beta-000081\build\XliffTasks.targets(90,5): error MSB4018:    at System.Xml.Linq.XDocument.Load(TextReader textReader, LoadOptions options) [D:\a\1\s\src\Microsoft.VisualStudio.Editors\Microsoft.VisualStudio.Editors.vbproj]
2019-01-18T12:58:14.2356293Z C:\Users\VssAdministrator\.nuget\packages\xlifftasks\0.2.0-beta-000081\build\XliffTasks.targets(90,5): error MSB4018:    at XliffTasks.Model.XlfDocument.Load(TextReader reader) [D:\a\1\s\src\Microsoft.VisualStudio.Editors\Microsoft.VisualStudio.Editors.vbproj]
2019-01-18T12:58:14.2356364Z C:\Users\VssAdministrator\.nuget\packages\xlifftasks\0.2.0-beta-000081\build\XliffTasks.targets(90,5): error MSB4018:    at XliffTasks.Model.Document.Load(Stream stream) [D:\a\1\s\src\Microsoft.VisualStudio.Editors\Microsoft.VisualStudio.Editors.vbproj]
2019-01-18T12:58:14.2356464Z C:\Users\VssAdministrator\.nuget\packages\xlifftasks\0.2.0-beta-000081\build\XliffTasks.targets(90,5): error MSB4018:    at XliffTasks.Model.Document.Load(String path) [D:\a\1\s\src\Microsoft.VisualStudio.Editors\Microsoft.VisualStudio.Editors.vbproj]
2019-01-18T12:58:14.2356538Z C:\Users\VssAdministrator\.nuget\packages\xlifftasks\0.2.0-beta-000081\build\XliffTasks.targets(90,5): error MSB4018:    at XliffTasks.Tasks.XlfTask.LoadXlfDocument(String path, String language, Boolean createIfNonExistent) [D:\a\1\s\src\Microsoft.VisualStudio.Editors\Microsoft.VisualStudio.Editors.vbproj]
2019-01-18T12:58:14.2356634Z C:\Users\VssAdministrator\.nuget\packages\xlifftasks\0.2.0-beta-000081\build\XliffTasks.targets(90,5): error MSB4018:    at XliffTasks.Tasks.UpdateXlf.ExecuteCore() [D:\a\1\s\src\Microsoft.VisualStudio.Editors\Microsoft.VisualStudio.Editors.vbproj]
2019-01-18T12:58:14.2356708Z C:\Users\VssAdministrator\.nuget\packages\xlifftasks\0.2.0-beta-000081\build\XliffTasks.targets(90,5): error MSB4018:    at XliffTasks.Tasks.XlfTask.Execute() [D:\a\1\s\src\Microsoft.VisualStudio.Editors\Microsoft.VisualStudio.Editors.vbproj]
2019-01-18T12:58:14.2356805Z C:\Users\VssAdministrator\.nuget\packages\xlifftasks\0.2.0-beta-000081\build\XliffTasks.targets(90,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute() [D:\a\1\s\src\Microsoft.VisualStudio.Editors\Microsoft.VisualStudio.Editors.vbproj]
2019-01-18T12:58:14.2356899Z C:\Users\VssAdministrator\.nuget\packages\xlifftasks\0.2.0-beta-000081\build\XliffTasks.targets(90,5): error MSB4018:    at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext() [D:\a\1\s\src\Microsoft.VisualStudio.Editors\Microsoft.VisualStudio.Editors.vbproj]
```

**Comment by janbaltus** [comment]
@davkean thank you!
We updated the PR with fixed file.

File: src/Microsoft.VisualStudio.Editors/PropPages/xlf/ReferencePropPage.zh-Hant.xlf [hunk 1]
```diff
@@ -14,7 +14,7 @@
       </trans-unit>
       <trans-unit id="ColHdr_Type.Text">
         <source>Type</source>
-        <target state="translated">類型</target>
+        <target state="translated">型別</target>
```
Comment (Varorbc) [inline src/Microsoft.VisualStudio.Editors/PropPages/xlf/ReferencePropPage.zh-Hant.xlf]: 型别是个什么鬼

File: src/Microsoft.VisualStudio.Editors/PropPages/xlf/ReferencePropPage.cs.xlf [hunk 2]
```diff
@@ -29,7 +29,7 @@
       </trans-unit>
       <trans-unit id="ColHdr_Path.Text">
         <source>Path</source>
-        <target state="translated">Cesta</target>
+        <target state="translated">PATH</target>
```
Comment (davkean) [inline src/Microsoft.VisualStudio.Editors/PropPages/xlf/ReferencePropPage.cs.xlf]: Why was this change made?

File: src/Microsoft.VisualStudio.Editors/PropPages/xlf/ReferencePropPage.es.xlf [hunk 3]
```diff
@@ -29,7 +29,7 @@
       </trans-unit>
       <trans-unit id="ColHdr_Path.Text">
         <source>Path</source>
-        <target state="translated">Ruta de acceso</target>
+        <target state="translated">PATH</target>
         <note />
```
Comment (davkean) [inline src/Microsoft.VisualStudio.Editors/PropPages/xlf/ReferencePropPage.es.xlf]: And this one?

**Comment by jmarolf** [comment]
This build is still failing with this error:

```
2019-01-21T13:17:32.3533777Z C:\Users\VssAdministrator\.nuget\packages\xlifftasks\0.2.0-beta-000081\build\XliffTasks.targets(90,5): error : 'ProjectSystem\Rules\xlf\PackageReference.xaml.cs.xlf' is out-of-date with 'ProjectSystem\Rules\PackageReference.xaml'. Run `msbuild /t:UpdateXlf` to update .xlf files or set UpdateXlfOnBuild=true to update them on every build, but note that it is strongly discouraged to set UpdateXlfOnBuild=true in official/CI build environments as they should not modify source code during the build. [D:\a\1\s\src\Microsoft.VisualStudio.ProjectSystem.Managed\Microsoft.VisualStudio.ProjectSystem.Managed.csproj]
2019-01-21T13:17:32.3534153Z     0 Warning(s)
2019-01-21T13:17:32.3534422Z     1 Error(s)
2019-01-21T13:17:32.3534601Z 
```

it looks like whatever tool updated `PackageReference.xaml.cs.xlf` did so incorrectly

File: src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/xlf/PackageReference.xaml.de.xlf [hunk 4]
```diff
@@ -77,16 +77,6 @@
         <target state="translated">Speicherort des Pakets, auf das verwiesen wird.</target>
         <note />
       </trans-unit>
-      <trans-unit id="BoolProperty|GeneratePathProperty|DisplayName">
-        <source>Generate path property</source>
-        <target state="new">Generate path property</target>
-        <note />
-      </trans-unit>
```
Comment (janbaltus) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/xlf/PackageReference.xaml.de.xlf]: This part of file was updated between moment we took xlf and localized id till moment will created PR.
Please merge file.

Comment (janbaltus) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/xlf/PackageReference.xaml.de.xlf]: @jmarolf this PR updated xlf file https://github.com/dotnet/project-system/commit/df5840152cca05183bff5d5d10c060a081ed05c2#diff-c550a87f8230c13f67d13c3ddfa142bb

**Comment by janbaltus** [comment]
@davkean PR updated with new version of xlf file and also implemented highlighted issues.

File: src/Microsoft.VisualStudio.Editors/MyExtensibility/xlf/AddMyExtensionsDialog.fr.xlf [hunk 5]
```diff
@@ -14,7 +14,7 @@
       </trans-unit>
       <trans-unit id="colHeaderExtensionName.Text">
         <source>Name</source>
-        <target state="translated">Nom</target>
+        <target state="translated">Nom </target>
```
Comment (drewnoakes) [inline src/Microsoft.VisualStudio.Editors/MyExtensibility/xlf/AddMyExtensionsDialog.fr.xlf]: This whitespace looks suspect. Extra spaces might cause layout problems.

Is there ever a case where we wouldn't want translated strings to be trimmed?

File: src/Microsoft.VisualStudio.Editors/MyExtensibility/xlf/AddMyExtensionsDialog.tr.xlf [hunk 6]
```diff
@@ -19,7 +19,7 @@
       </trans-unit>
       <trans-unit id="colHeaderExtensionVersion.Text">
         <source>Version</source>
-        <target state="translated">Sürüm</target>
+        <target state="translated">SÜRÜM</target>
```
Comment (drewnoakes) [inline src/Microsoft.VisualStudio.Editors/MyExtensibility/xlf/AddMyExtensionsDialog.tr.xlf]: Should the be capitalised? The English form is not.

File: src/Microsoft.VisualStudio.Editors/MyExtensibility/xlf/MyExtensibilityPropPage.de.xlf [hunk 7]
```diff
@@ -29,7 +29,7 @@
       </trans-unit>
       <trans-unit id="colHeaderExtensionVersion.HeaderText">
         <source>Version</source>
-        <target state="translated">Version</target>
+        <target state="translated">VERSION</target>
```
Comment (drewnoakes) [inline src/Microsoft.VisualStudio.Editors/MyExtensibility/xlf/MyExtensibilityPropPage.de.xlf]: Another capitalisation. There are more examples of capitalisation changes and white space changes throughout.

File: src/Microsoft.VisualStudio.Editors/OptionPages/xlf/GeneralOptionPageResources.de.xlf [hunk 8]
```diff
@@ -19,7 +19,7 @@
       </trans-unit>
       <trans-unit id="General_FastUpToDateCheck_LogLevel_None">
         <source>None</source>
-        <target state="translated">Ohne</target>
+        <target state="translated">NONE</target>
```
Comment (drewnoakes) [inline src/Microsoft.VisualStudio.Editors/OptionPages/xlf/GeneralOptionPageResources.de.xlf]: It appears the translation was undone here.

File: src/Microsoft.VisualStudio.Editors/OptionPages/xlf/GeneralOptionPageResources.es.xlf [hunk 9]
```diff
@@ -14,12 +14,12 @@
...
       <trans-unit id="General_FastUpToDateCheck_LogLevel_Info">
         <source>Info</source>
-        <target state="translated">Info</target>
+        <target state="translated">Información</target>
         <note />
       </trans-unit>
       <trans-unit id="General_FastUpToDateCheck_LogLevel_None">
         <source>None</source>
-        <target state="translated">Ninguno</target>
+        <target state="translated">NONE</target>
```
Comment (drewnoakes) [inline src/Microsoft.VisualStudio.Editors/OptionPages/xlf/GeneralOptionPageResources.es.xlf]: Translation also undone here.

File: src/Microsoft.VisualStudio.Editors/OptionPages/xlf/GeneralOptionPageResources.it.xlf [hunk 10]
```diff
@@ -14,7 +14,7 @@
       </trans-unit>
       <trans-unit id="General_FastUpToDateCheck_LogLevel_Info">
         <source>Info</source>
-        <target state="translated">Informazioni</target>
+        <target state="translated">Info</target>
```
Comment (drewnoakes) [inline src/Microsoft.VisualStudio.Editors/OptionPages/xlf/GeneralOptionPageResources.it.xlf]: Some changes to this string are converting the short form "Info" to the long translated form, others are shortening. Seems inconsistent.

File: src/Microsoft.VisualStudio.Editors/OptionPages/xlf/GeneralOptionPageResources.ko.xlf [hunk 11]
```diff
@@ -34,7 +34,7 @@
       </trans-unit>
       <trans-unit id="General_UsePreviewSdk">
         <source>Use previews of the .NET Core SDK</source>
-        <target state="translated">.NET Core SDK의 미리 보기 사용</target>
+        <target state="translated">.NET Core SDK</target>
```
Comment (drewnoakes) [inline src/Microsoft.VisualStudio.Editors/OptionPages/xlf/GeneralOptionPageResources.ko.xlf]: This removes "의 미리 보기 사용" which translates as "Use preview of", which seems like an important part of this string.

File: src/Microsoft.VisualStudio.Editors/OptionPages/xlf/GeneralOptionPageResources.tr.xlf [hunk 12]
```diff
@@ -19,7 +19,7 @@
       </trans-unit>
       <trans-unit id="General_FastUpToDateCheck_LogLevel_None">
         <source>None</source>
-        <target state="translated">Yok</target>
+        <target state="translated">yok</target>
```
Comment (drewnoakes) [inline src/Microsoft.VisualStudio.Editors/OptionPages/xlf/GeneralOptionPageResources.tr.xlf]: Converted to lower case, which looks wrong. It seems like all translations of this key have been incorrect.

File: src/Microsoft.VisualStudio.Editors/PropPages/xlf/AdvCompilerSettingsPropPage.zh-Hant.xlf [hunk 13]
```diff
@@ -49,7 +49,7 @@
       </trans-unit>
       <trans-unit id="CustomConstantsExampleLabel.Text">
         <source>Example: Name1="Value1",Name2="Value2",Name3="Value3"</source>
-        <target state="translated">範例:Name1="Value1",Name2="Value2",Name3="Value3"</target>
+        <target state="translated">範例: Name1 ="Value1"，Name2 ="Value2"，Name3 = Value3"</target>
```
Comment (drewnoakes) [inline src/Microsoft.VisualStudio.Editors/PropPages/xlf/AdvCompilerSettingsPropPage.zh-Hant.xlf]: Extra spaces around the `=` chars may be incorrect.

File: src/Microsoft.VisualStudio.Editors/PropPages/xlf/ApplicationPropPage.fr.xlf [hunk 14]
```diff
@@ -54,7 +54,7 @@
       </trans-unit>
       <trans-unit id="AppIconBrowse.Text">
         <source>Browse...</source>
-        <target state="translated">Parcourir...</target>
+        <target state="translated">Parcourir</target>
```
Comment (drewnoakes) [inline src/Microsoft.VisualStudio.Editors/PropPages/xlf/ApplicationPropPage.fr.xlf]: Should not remove ellipsis.

**Comment by drewnoakes** [comment]
I stopped reviewing around 20% of the way in. There are many more instances of the issues highlighted within that 20%, and likely more within the remaining unreviewed 80%.

**Comment by jmarolf** [comment]
and AddImports is still not up to date
```
2019-01-22T11:57:37.8291892Z C:\Users\VssAdministrator\.nuget\packages\xlifftasks\0.2.0-beta-000081\build\XliffTasks.targets(90,5): error : 'AddImportsDialogs\xlf\AddImports.cs.xlf' is out-of-date with 'AddImportsDialogs\AddImports.resx'. Run `msbuild /t:UpdateXlf` to update .xlf files or set UpdateXlfOnBuild=true to update them on every build, but note that it is strongly discouraged to set UpdateXlfOnBuild=true in official/CI build environments as they should not modify source code during the build. [D:\a\1\s\src\Microsoft.VisualStudio.Editors\Microsoft.VisualStudio.Editors.vbproj]
2019-01-22T11:57:37.8292277Z     0 Warning(s)
```

**Comment by tmeschter** [comment]
@janbaltus Should we be concerned about the already-translated strings that seem to have been reverted to untranslated text, or accept them as-is for now and trust that a future PR will update them as necessary?

File: src/Microsoft.VisualStudio.Editors/AddImportsDialogs/xlf/AddImports.cs.xlf [hunk 15]
```diff
@@ -29,11 +27,8 @@
         <note>
       Defines the replacement string for use in the "Qualify Current Line" option of the Add Imports collision dialog. 
       This text will be formated as richtext. 
-      
       See the comments for AddImportsMainTextFormatString for more details about the richt text formating codes
       contained in this resource.
-
-
```
Comment (drewnoakes) [inline src/Microsoft.VisualStudio.Editors/AddImportsDialogs/xlf/AddImports.cs.xlf]: Why were these lines removed? Does this matter?

File: src/Microsoft.VisualStudio.AppDesigner/Resources/xlf/Designer.pt-BR.xlf [hunk 16]
```diff
@@ -265,12 +265,12 @@ Clique em OK para ignorar e continuar. Clique em CANCELAR para escolher um camin
       </trans-unit>
       <trans-unit id="PPG_PostBuildCommandLineTitle">
         <source>Post-build Event Command Line</source>
-        <target state="translated">Linha de Comando de Eventos Pós-build</target>
+        <target state="translated">Linha de comando do evento de pós-compilação</target>
```
Comment (drewnoakes) [inline src/Microsoft.VisualStudio.AppDesigner/Resources/xlf/Designer.pt-BR.xlf]: Changes capitalisation.

File: src/Microsoft.VisualStudio.Editors/PropPages/xlf/DebugPropPage.pt-BR.xlf [hunk 17]
```diff
@@ -4,7 +4,7 @@
     <body>
       <trans-unit id="StartActionLabel.Text">
         <source>Start action</source>
-        <target state="translated">Iniciar ação</target>
+        <target state="translated">Iniciar Ação</target>
```
Comment (drewnoakes) [inline src/Microsoft.VisualStudio.Editors/PropPages/xlf/DebugPropPage.pt-BR.xlf]: Changes capitalisation.

File: src/Microsoft.VisualStudio.Editors/PropPages/xlf/PackagePropPage.zh-Hans.xlf [hunk 18]
```diff
@@ -69,52 +69,52 @@
...
       <trans-unit id="PackageRequireLicenseAcceptance.Text">
         <source>Require &amp;license acceptance</source>
-        <target state="translated">需要和许可证接受 (&amp;L)</target>
+        <target state="translated">要求接受许可证(&amp;L)</target>
         <note />
       </trans-unit>
       <trans-unit id="PackageIdLabel.Text">
         <source>Package &amp;id:</source>
-        <target state="translated">包 ID (&amp;I):</target>
+        <target state="translated">包 ID(&amp;I)</target>
```
Comment (drewnoakes) [inline src/Microsoft.VisualStudio.Editors/PropPages/xlf/PackagePropPage.zh-Hans.xlf]: `:` removed.

File: src/Microsoft.VisualStudio.Editors/PropPages/xlf/ApplicationPropPage.fr.xlf [hunk 19]
```diff
@@ -14,7 +14,7 @@
       </trans-unit>
       <trans-unit id="OutputTypeLabel.Text">
         <source>O&amp;utput type:</source>
-        <target state="translated">&amp;Type de sortie :</target>
+        <target state="translated">T&amp;ype de sortie :</target>
```
[the remark this answers is not in the document]
Comment (janbaltus) [reply src/Microsoft.VisualStudio.Editors/PropPages/xlf/ApplicationPropPage.fr.xlf]: This is correct for French. They add space before :

File: src/Microsoft.VisualStudio.Editors/PropPages/xlf/DebugPropPage.zh-Hans.xlf [hunk 20]
```diff
@@ -49,7 +49,7 @@
       </trans-unit>
       <trans-unit id="CommandLineArgsLabel.Text">
         <source>Comma&amp;nd line arguments:</source>
-        <target state="translated">命令行参数(&amp;N): </target>
+        <target state="translated">命令行参数(&amp;N)</target>
```
Comment (drewnoakes) [inline src/Microsoft.VisualStudio.Editors/PropPages/xlf/DebugPropPage.zh-Hans.xlf]: Removed ':'.

File: src/Microsoft.VisualStudio.Editors/PropPages/xlf/ApplicationPropPage.fr.xlf (hunk 19 above)
[the remark this answers is not in the document]
Comment (drewnoakes) [reply src/Microsoft.VisualStudio.Editors/PropPages/xlf/ApplicationPropPage.fr.xlf]: TIL the French place a space before punctuation marks having an even number of marks (`;:!?`).

https://www.quora.com/Why-do-French-speakers-place-a-space-before-exclamation-points

**Comment by tmeschter** [comment]
Closing in favor of #4516.
